packages/vsce/src/commands/enableResourceCommand.ts

```diff
@@ -61,37 +60,38 @@ export const getEnableResourceCommands = (tree: CICSTree, treeview: TreeView<CIC
   const enableBundle = async (treeNode: CICSResourceContainerNode<IBundle>) => {
     const nodes = findSelectedNodes(treeview, BundleMeta, treeNode);
     if (!nodes || !nodes.length) {
-      await window.showErrorMessage("No CICS Bundles selected");
+      await window.showErrorMessage(l10n.t("No CICS Bundles selected"));
       return;
     }
 
     await actionTreeItem({
       action: "ENABLE",
       nodes,
       tree,
-      pollCriteria: (response) => response.records?.cicsbundle?.enablestatus.toUpperCase() === "ENABLED"
+      pollCriteria: (response) => response.records?.cicsbundle?.enablestatus.toUpperCase() === "ENABLED",
     });
   };
 
   const enableJVMServer = async (treeNode: CICSResourceContainerNode<IJVMServer>) => {
     const nodes = findSelectedNodes(treeview, JVMServerMeta, treeNode);
     if (!nodes || !nodes.length) {
-      await window.showErrorMessage("No CICS JVM server selected");
+      await window.showErrorMessage(l10n.t("No CICS JVM server selected"));
       return;
     }
 
     await actionTreeItem({
       action: "ENABLE",
       nodes,
+
       tree,
-      pollCriteria: (response) => response.records?.cicsjvmserver?.enablestatus.toUpperCase() === "ENABLED"
+      pollCriteria: (response) => response.records?.cicsjvmserver?.enablestatus.toUpperCase() === "ENABLED",
     });
   };
 
   const enableJVMEndpoint = async (treeNode: CICSResourceContainerNode<IJVMEndpoint>) => {
     const nodes = findSelectedNodes(treeview, JVMEndpointMeta, treeNode);
     if (!nodes || !nodes.length) {
-      await window.showErrorMessage("No CICS JVMEndpoint selected");
+      await window.showErrorMessage(l10n.t("No CICS JVMEndpoint selected"));
```

Comment:
While we're here I think we could fix the words slightly. I don't think we'll see this message but let's fix it.

```suggestion
      await window.showErrorMessage(l10n.t("No CICS JVM endpoint selected"));
```